app.py

```diff
@@ -117,7 +118,7 @@ def update_criterion(id_):
 
     category_id = data.get('category_id')
     if category_id and category_id != criterion.category_id:
-      return jsonify(text=strings.criterion_cannot_change_category), 400
+      return jsonify(text=strings.cannot_change_category), 400
```

Comment:
@brittanylau I changed this string from `criterion_cannot_change_category` to `cannot_change_category` because a few models have a relation to categories. Let me know if that's okay!

Reply:
Yes of course!!! I like it :)